package.json

```diff
@@ -2,7 +2,11 @@
   "name": "@quasar/quasar-app-extension-apollo",
   "version": "2.0.0-beta.1",
   "description": "A Quasar app extension to add GraphQL support using Apollo Client.",
-  "keywords": ["Quasar", "GraphQL", "Apollo"],
+  "keywords": [
+    "quasar",
+    "graphql",
+    "apollo"
+  ],
   "contributors": [
     "Scott Molinari <scott.molinari@adduco.de>",
     "Ejez <ejez@protonmail.ch>"
```

Comment:
Could you add your name here too please @IlCallo. :)

Scott

Reply:
Will do, but only because you asked 🤣